BC-NOV-20-Proyectos/alerta-covid-web-team02
# Iss23 hybrid auth

# Description
What did you implemented/Why did you implemented this:
 Some fixes were made in the api authorzation module, because when a user log in when he already do it,
devise responded with the default html. So a method that check if the user is already signed was modified to
respond with a json indicating this.
 
 # Testing
- Download the project.
- Run the rails project.
- Enter to the admin with the email: admin@example.com and the password: "password" (ignore the insecurity of the credentials, it is only for testing).
- Create a user in the admin, after this you will receive in your email a password.
- Make the next petition: 
 POST http://localhost:3000/api/user/sign_in?api_user[email]=email@main.com&api_user[password]=yourkey
You have to replace the credentials with yours.
- You should have a responde with your data and your token.
- If you try to login again you should receive a json indicating this and with a new token.
